packages/react-scripts/template/.storybook/webpack.config.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        loaders: ['style-loader', 'css-loader', 'sass-loader'],
+        include: path.resolve(__dirname, '../'),
+      },
+    ],
+  },
+};
```

Comment:
There are some recent changes I made to the similar file in [@ifit/omnius-webview](https://github.com/ifit/omnius-webview) because it wasn't processing the CSS files with the postcss autoprefixer. May need to add those options here as well.

Reply:
Thanks for that. I've made those changes